app/backend/src/couchers/materialized_views.py

```diff
@@ -304,7 +322,7 @@ class UserResponseRate(Base):
     __table__ = user_response_rates
 
 
-def refresh_materialized_views(payload: empty_pb2.Empty):
+def refresh_materialized_views(payload: empty_pb2.Empty) -> None:
```

Comment:
What's the diff between `-> None` and `-> NoReturn`?

Reply:
NoReturn means that the function will always raise an exception (e.g., abort())